src/components/updatemenus/attributes.js

```diff
@@ -11,6 +11,7 @@
 var fontAttrs = require('../../plots/font_attributes');
 var colorAttrs = require('../color/attributes');
 var extendFlat = require('../../lib/extend').extendFlat;
+var padAttrs = require('../../plots/pad_attributes');
```

Comment:
Agreed. Font attributes would probably go with it? At least internal reorg is easy refactoring, so can try to cut down on `plots/` bit by bit (pun?)